decidim-proposals/app/controllers/decidim/proposals/proposals_controller.rb

```diff
@@ -87,6 +87,10 @@ def default_filter_params
           related_to: ""
         }
       end
+
+      def check_proposal_limit!
+        raise "Proposal limit reached" if proposal_limit_reached?
```

Comment:
`raise`? Can we make it redirect to the index with a flash message or something?